### ISSUE TITLE
Add sortable indicator for columns that are sortable, but sorting is not applied.

### ISSUE THREAD

**Issue by aleksandarevangelatov** [telerik/kendo-ui-core#7270]
### Feature Request

Currently, an indicator for columns which are sortable is missing if the column is not sorted. An indicator/icon and the relative accessibility attributes should be added

### Environment

* **Kendo UI version:** 2023.1.314
* **jQuery version:** x.y
* **Browser:** [all]
